src/synology_dsm/api/photos/model.py

```diff
@@ -21,3 +21,4 @@ class SynoPhotosItem:
     file_size: str
     thumbnail_cache_key: str
     thumbnail_size: str
+    owner_user_id: int
```

Comment:
i think we shoud add a bool like `is_shared` which is set to True when "owner_user_id == 0"